polaris/polaris/management/commands/testnet.py

```diff
@@ -178,15 +179,19 @@ def issue(self, **options):
     def add_balance(self, code, amount, accounts, dest, src, issuer):
         tb = TransactionBuilder(
             self.account_from_json(accounts[src.public_key]),
-            base_fee=500,
+            base_fee=settings.MAX_TRANSACTION_FEE_STROOPS
+            or settings.HORIZON_SERVER.fetch_base_fee(),
             network_passphrase="Test SDF Network ; September 2015",
         )
-        payment_amount = None
         balance = self.get_balance(code, accounts[dest.public_key])
         if not balance:
             print(f"\nCreating {code} trustline for {dest.public_key}")
-            limit = amount if src == issuer else None
-            tb.append_change_trust_op(code, issuer.public_key, limit, dest.public_key)
+            if settings.MAX_TRANSACTION_FEE_STROOPS:
+                # halve the base_fee because there are 2 operations
```

Comment:
I think this is a gap in my Stellar knowledge, why do we halve the base fee (for example basefee was 100 is now 50) for 2 operations?